src/Backend/Modules/Tags/Entity/Tag.php

```diff
@@ -0,0 +1,208 @@
+<?php
+
+namespace Backend\Modules\Tags\Entity;
+
+/*
+ * This file is part of Fork CMS.
+ *
+ * For the full copyright and license information, please view the license
+ * file that was distributed with this source code.
+ */
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * This is the Tag Entity
+ *
+ * @author Jeroen Desloovere <info@jeroendesloovere.be>
+ *
+ * @ORM\Entity
+ * @ORM\Table(name="tags")
+ */
+class Tag
+{
+    /**
+     * @var int
+     *
+     * @ORM\Id
+     * @ORM\Column(type="integer", length=11)
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string", length=255)
+     */
+    private $language;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string", length=255)
+     */
+    private $tag;
```

Comment:
I would use 'name' and not 'tag', since you will store the 'name' property of the 'tag' object in here. This will make it less confusing I guess.